gobblin-runtime/src/main/java/org/apache/gobblin/runtime/Task.java

```diff
@@ -180,6 +182,7 @@ public Task(TaskContext context, TaskStateTracker taskStateTracker, TaskExecutor
     this.jobId = this.taskState.getJobId();
     this.taskId = this.taskState.getTaskId();
     this.taskKey = this.taskState.getTaskKey();
+    this.isIgnoreCloseFailures = this.taskState.isIgnoreCloseFailures();
```

Comment:
Instead of setting this in the `TaskState` where it is set in two places, why not set it here like `this.isIgnoreCloseFailures = this.taskState.getJobState().getProp(...)`?

Reply:
Thanks for the comment! Addressed it.